packages/deploy-tooling/src/base/validate.ts

```diff
@@ -99,7 +99,7 @@ export function formatSummary(summary: SummaryRecord[]): string {
                     statusSymbol = yellow('?');
                     break;
             }
-            return `${statusSymbol} ${next.message}`;
+            return `${'    '}${statusSymbol} ${next.message}`;
```

Comment:
```suggestion
            return `    ${statusSymbol} ${next.message}`;
```
Should the `${}` be removed here?